tests/torchtune/data/test_messages.py

```diff
@@ -479,6 +609,200 @@ def test_call_image_messages(self, mock_load_image):
         mock_load_image.assert_called_once_with("https://example.com")
 
 
+class TestAlpacaToMessages:
```

Comment:
What's the difference between this and the existing `TestAlpacaToMessages` [here](https://github.com/pytorch/torchtune/blob/67a8706abd993d4b03c70506075a2a9804919148/tests/torchtune/datasets/test_alpaca_dataset.py#L93)? They look fairly similar

Reply:
I did not know we had `TestAlpacaToMessages` in test_alpaca_dataset.py and added one in test_messages.py. 

Both are similar and the one in test_messages.py runs more tests. Please let me know if I can remove TestAlpacaToMessages from [here](https://github.com/pytorch/torchtune/blob/67a8706abd993d4b03c70506075a2a9804919148/tests/torchtune/datasets/test_alpaca_dataset.py#L93)